cms/templates/js/discussion-editor.underscore

```diff
@@ -0,0 +1,26 @@
+<form>
```

Comment:
I'm guessing this is adapted from the existing editor templates?

Are there any key details (besides the content itself) that are worth calling out?

I'll punt on reviewing this file for now, as it looks like it will change a bit, following conversation on the related PR (to remove some of these UI indicators); let me know if/when I should review this in detail.

Reply:
> I'm guessing this is adapted from the existing editor templates?
Are there any key details (besides the content itself) that are worth calling out?

Not really, you can just compare it to the [staff-lock-editor.underscore](https://github.com/edx/edx-platform/blob/1c90220462dfcc38096617c69796bb348289926d/cms/templates/js/staff-lock-editor.underscore), which I used as a template.

> I'll punt on reviewing this file for now, as it looks like it will change a bit, following conversation on the related PR (to remove some of these UI indicators); let me know if/when I should review this in detail.

Sure, will do.